etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -2,39 +2,42 @@
 
 set -eux
 
+if [[ ! -f $BASE_PATH/vault-pw ]]; then
```

Comment:
Reference to BASE_PATH before definition will blow up with `set -u`

Reply:
:facepalm: Fixed